spec/app/ics-028-cross-chain-validation/overview_and_basic_concepts.md

```diff
@@ -205,3 +213,53 @@ This approach is depicted in the following Figure that shows an overview of the
   The unbonding operation is completed later once the provider CCV module registered maturity notifications for `VSC2` from all consumer chains.
 
 ![Completion of Unbonding Operations](./figures/ccv-unbonding-overview.png?raw=true)
+
+### Consumer Initiated Slashing
+[&uparrow; Back to Outline](#outline)
+
+For the [Security Model](#security-model) to be preserved, misbehaving validators MUST be slashed (and MAY be jailed, i.e., removed from the validator set). 
+A prerequisite to slash validators is to receive valid evidence of their misbehavior. 
+Thus, when slashing a validator, we distinguish between three events and the heights when they occur:
+- `infractionHeight`, the height at which the misbehavior (or infraction) happened;
+- `evidenceHeight`, the height at which the evidence of misbehavior is received;
+- `slashingHeight`, the height at which the validator is slashed (and jailed). 
+
+> **Note**: In the context of single-chain validation, usually `evidenceHeight = slashingHeight`. 
+
+The [Security Model](#security-model) guarantees that any misbehaving validator can be slashed for at least the unbonding period, 
+i.e., as long as that validator's tokens are not unbonded yet, they can be slashed. 
+However, if the tokens start unbonding before `infractionHeight`, i.e., the tokens did not contribute to the voting power that committed the infraction, 
+then the tokens MUST NOT be slashed.
+
+In the context of CCV, validators (with tokens bonded on the provider chain) MUST be slashed for infractions committed on the consumer chains. 
+Thus, although the infractions are committed on the consumer chains and evidence of these infractions is submitted to the consumer chains, the slashing happens on the provider chain. As a result, the Consumer Initiated Slashing operation requires, for every consumer chain, a mapping from consumer chain block heights to provider chain block heights.
+The following figure shows an overview of how CCV creates such a mapping using VSC IDs.
+For clarity, we use `Hp*` and `Hc*` to denote block heights on the provider chain and consumer chain, respectively. 
+
+![Mapping Between Provider and Consumer Heights](./figures/ccv-height-mapping-overview.png?raw=true)
+
+- For every block, the provider CCV module maps the ID of the VSC it provides to the consumer chains to the height of the subsequent block, i.e., `VSCtoH(VSC.id) = Hp + 1`, for a VSC provided at height `Hp`. 
```

Comment:
I had a hard time understanding the notation here. To me it sounds like there is a map data structure of some sort looking like this:

```
{
  [VSC.id]: Hp + 1,
  ...
}
```

`VSCtoH(VSC.id) = Hp + 1` implies to me some kind of conversion or function call. Can we use square brackets like this? `VSCtoH[VSC.id] = Hp + 1` That is more in line with the map accessing syntax in most languages.